main.go

```diff
@@ -93,6 +94,8 @@ func main() {
 		}
 	} else if *tagsFlag != "" {
 		coverageCommand = "go test ./... -tags \"" + *tagsFlag + "\" -coverprofile=coverage.out && go tool cover -func=coverage.out"
+	} else if *shortFlag {
```

Comment:
Should this -short flag be added via `else if` or might it be better to tack it on to the end of a growing command?
ie: `coverageCommand = coverageCommand + " -short"` or something like that

Otherwise, would need some sort of documentation/notification that -tags will over-ride short above in the flag parser.